Contactpage form

Tobias-hubs opened the pull request:
**Description**
Contact page with a structured form using Material UI. 
Follows the same styling principles as our exisiting Kino project. 

**Features**
Responsive and styled contact form. Using theme.ts as well as ThemeProvider to match styling. 
- Implemented contact page component. 
- Built reusable FormFiled component for input fields
- Applied MUI styling to match the projects theme.
- Basic Vaildation logic

**Testing**
Manual testing

**Preview**

![image](https://github.com/user-attachments/assets/2866779b-df99-478d-b03c-a34b59c81d0a)

nilsishome reviewed (changes requested):
Great work with implementing the UI for contact page, Tobias!

Maybe you don't have to import our theme directly to your page.tsx file, because we already got it in app/layout.tsx.
And is it possible to change the background color of the "Skicka" button, to make it more user friendly?

It looks otherwise almost exactly like you did it in our previous project. Nice implementation of MUI by the way! Keep up the good work! :+1:

Tobias-hubs commented:
> Great work with implementing the UI for contact page, Tobias!
> 
> Maybe you don't have to import our theme directly to your page.tsx file, because we already got it in app/layout.tsx. And is it possible to change the background color of the "Skicka" button, to make it more user friendly?
> 
> It looks otherwise almost exactly like you did it in our previous project. Nice implementation of MUI by the way! Keep up the good work! 👍

Thanks for great feedback Nils, You are right it was unnecessary to import theme and use themeprovider as we have our global theme. 
This is now fixed in my last Fix: commit. Also changed the button color and position.

nilsishome reviewed (approved):
It looks great! Good job! :raised_hands:

MagnusTerak reviewed (approved):
Well done Tobias! This looks good and matches our previous contact page well!
